src/GafferImage/Catalogue.cpp

```diff
@@ -1059,15 +1063,5 @@ void Catalogue::compute( ValuePlug *output, const Context *context ) const
 
 void Catalogue::computeNameToIndexMapping()
 {
-	CompoundDataPtr mapData = new CompoundData();
-	std::map<InternedString, DataPtr> &map = mapData->writable();
-
-	int count = 0;
-	for( const auto &image : imagesPlug()->children() )
-	{
-		map[image->getName()] = new IntData( count );
-		++count;
-	}
-
-	mappingPlug()->setValue( mapData );
+	// TODO: Remove, along with `mappingPlug` when merging forwards beyond a patch release
```

Comment:
Removing a private plug and/or a private method isn't a breaking change, so we should just do this now.